docs/Gemfile.lock

```diff
@@ -0,0 +1,299 @@
+GEM
```

Comment:
Do we need this file?

Reply:
Looks like it is needed -- https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore